### ISSUE TITLE
Allow Universal Block Remover to detect Block Mines / Wire Cutters bug (?)

### ISSUE THREAD

**Issue by MartinSalinas98** [Geforce132/SecurityCraft#144]
I'd like to have an option on the config file to allow universal object remover to detect disguised mines (p. ex. Dirt Mine) when right clicking on them so when I'm not the owner it displays the same error message as if it was a reinforced block.

This is because, playing in a survival server, it is impossible to know for sure if a block is a disguised mine or not until it explodes, and, even if I suspect that the block might be a mine, I have no way of checking it without the admin tool using creative mode.

I also thought that the Wire Cutters might do the trick if I just right clicked on them, and that works for the normal mine, but when it is a disguised mine the Wire Cutters do nothing (I'm not sure if it's a bug or it's intended to be like that).

And, even if that was a bug and the wire cutter could defuse the disguised mine allowing me to break it, since it doesn't display any message when doing it, I have no way of knowing if I just right clicked a mine or a vanilla block, and that's a problem if you need to check a big area block by block, right clicking and then breaking every single one of them.

TLDR: my feature request + possible bug report are:

- Wire Cutters can´t defuse disguised mines (not sure if it's meant to be like that)

- Add option to config file so Universal Block Remover displays same error message when right clicking on a disguised mine not owned by you (I though of it as an option because somebody might actually like not being able to detect the mine with the remover).

- Wire cutters might as well send a message when a disguised mine has been defused so the player knows it's safe to break it (since with that kind of mines you can't see the green light normal mines show).

**Comment by bl4ckscor3** [comment]
Thank you for the suggestion, however we do not want a possibility for the block mines to be detectable in any way. Sorry.
